### PR TITLE
fix(android): check for root view on configuration changed

### PR DESCRIPTION
**Pull request by vchimev**
- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] You have signed the [CLA](http://www.nativescript.org/cla).
- [x] All existing tests are passing: https://github.com/NativeScript/NativeScript/blob/master/DevelopmentWorkflow.md#running-unit-tests.
- [ ] Tests for the changes are included - https://github.com/NativeScript/NativeScript/blob/master/WritingUnitTests.md.

A Playground and Preview apps scenario.

Relates to https://github.com/NativeScript/NativeScript/issues/7800.

### CODE REVIEW

**Comment by vchimev** [comment]
test --ignore

File: tns-core-modules/application/application.android.ts [hunk 1]
```diff
@@ -413,11 +413,14 @@ function initComponentCallbacks() {
         }),
 
         onConfigurationChanged: profile("onConfigurationChanged", function (newConfiguration: android.content.res.Configuration) {
+            const rootView = getRootView();
             const newOrientation = getOrientationValue(newConfiguration);
 
             if (androidApp.orientation !== newOrientation) {
                 androidApp.orientation = newOrientation;
-                orientationChanged(getRootView(), newOrientation);
+                if (rootView) {
```
Comment (manoldonev) [inline tns-core-modules/application/application.android.ts]: We have those checks on at 3-4 places now -- I would move it in the beginning of `orientationChanged` and `systemAppearanceChanged` and simply return if null. In this way we ensure that should we call the same methods elsewhere, we will not need to remember that the check should be added as well.

**Comment by manoldonev** [comment]
test

**Comment by manoldonev** [comment]
test --ignore

**Comment by manoldonev** [comment]
test --ignore

**Comment by manoldonev** [comment]
test --ignore

**Comment by manoldonev** [comment]
test --ignore

**Pull request merged**